templates/zerver/help/configure-authentication-methods.md

```diff
@@ -0,0 +1,27 @@
+# Configure authentication methods
+
+{!admin-only.md!}
+
+{!go-to-the.md!} [Authentication methods](/#organization/auth-methods)
+{!admin.md!}
+
+2. Enable or disable the authentication methods you want.
```

Comment:
May want to rephrase this to something that's more explicit, such as "Toggle the checkboxes next to the following options to configure your organization's authentication methods:"